Gemfile

```diff
@@ -21,4 +21,5 @@ end
 
 group :test do
   gem 'honeybadger', '3.3.1-1', git: 'git@github.com:Invoca/honeybadger-ruby', ref: 'bb5f2b8a86e4147c38a6270d39ad610fab4dd5e6'
+  gem 'invoca-utils', path: '../invoca-utils'
```

Comment:
I believe our best practice is to no longer pin gems in the gemfile that are in the gemspec. Can you remove these from above?
```
gem 'actionmailer',  '>= 4.2.11.1'
gem 'actionpack',    '>= 4.2.11.1'
gem 'activesupport', '>= 4.2.11.1'
```

Reply:
done